cmake/compilerDefinitions.cmake

```diff
@@ -1,10 +1,20 @@
-if (UNIX)
+if (MSVC)
+    # Visual Studio only sets _DEBUG
     if (CMAKE_BUILD_TYPE MATCHES "Debug")
+        add_definitions(-DDEBUG)
+    endif()
+endif()
+if (UNIX)
+    # TODO: check if this can be enabled again for Clang - also done in Makefile
+    if (CMAKE_BUILD_TYPE MATCHES "Debug" AND NOT (CMAKE_CXX_COMPILER_ID MATCHES "Clang"))
         add_definitions(-D_GLIBCXX_DEBUG)
     endif()
     if (HAVE_RULES)
         add_definitions(-DHAVE_RULES -DTIXML_USE_STL)
     endif()
+    if (USE_Z3)
```

Comment:
Is there a special reason, why this is under `if (UNIX)`?

Reply:
Probably not. Taking baby steps with cleaning this up.

Reply:
moved this out of this block